.github/workflows/test.yml

```diff
@@ -137,6 +129,89 @@ jobs:
         RIDDLE_TOKEN: ${{ secrets.RIDDLE_TOKEN }}
       run: test-drupal-project run_tests
 
+  test-upgrade:
+
+    runs-on: ubuntu-latest
+
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'test-upgrade') || github.event_name == 'schedule' }}
+
+    env:
+      DRUPAL_TESTING_TEST_DEPRECATION: false
+      DRUPAL_TESTING_COMPOSER_PROJECT_VERSION: 2.x-dev
+      DRUPAL_TESTING_COMPOSER_PROJECT: BurdaMagazinOrg/thunder-project
+      DRUPAL_TESTING_TEST_GROUP: "Thunder --exclude-group NoUpdate"
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          coverage: none
+          php-version: '7.3'
+          extensions: Imagick
+
+      - name: Setup problem matchers for PHP
+        run: echo "::add-matcher::${{ runner.tool_cache }}/php.json"
+
+      - name: Setup problem matchers for PHPUnit
+        run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+      - name: Start MySql service
+        run: sudo /etc/init.d/mysql start
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache
+          key: ${{ runner.os }}-composer-cache-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-cache-
+
+      - name: Get build environment
+        run: composer global require thunder/drupal-testing
+
+      - name: Prepare the build
+        run: test-drupal-project prepare_build
+
+      - name: Use Thunder 2 lockfile
+        run: |
+          touch /tmp/test/thunder/finished-stages/build
+          cp ${GITHUB_WORKSPACE}/tests/fixtures/thunder2.composer.lock composer.lock
+          composer install
+          composer drupal-scaffold
+        working-directory: /tmp/test/thunder/install
+
+      - name: Install drupal
+        run: test-drupal-project install
+
+      - name: Update the docroot
+        run: |
+          composer remove thunder/testsite_builder --dev --no-update
+          composer require drupal/paragraphs:1.10 drupal/core:~8.7.0 --no-update
+          composer update
+          vendor/bin/drush updb -y
+          composer remove drupal/paragraphs --no-update
+          composer update
+          composer require burdamagazinorg/thunder:~8.2.51 --no-update
+          composer update
+          vendor/bin/drush updb -y
+          composer remove burdamagazinorg/thunder
+          composer require "thunder/thunder-distribution:*" drupal/core:~8.9.0 --no-update
+          composer update
+          composer require "thunder/thunder_testing_demo:3.x-dev" "drupal/media_entity:^2.0-beta4" "drupal/video_embed_field:^2.2" "drupal/media_entity_image" "drupal/riddle_marketplace:^3.0-beta2"
+          vendor/bin/drush updb -y
+          vendor/bin/drush en vem_migrate_oembed -y
+          vendor/bin/drush vem:migrate_oembed
+          vendor/bin/drush cr
+          cd docroot
+          php core/scripts/db-tools.php dump-database-d8-mysql > dump.sql
```

Comment:
I guess this is not a sql file, also where is it used?

Reply:
Right, it's a php file and it's used here 
https://github.com/thunder/drupal-testing/blob/master/lib/stages/run_tests.sh#L23